### PR TITLE
Google login fix

### PR DESCRIPTION
**Pull request by aydevelop**
Please use Binary's GoogleKey because my works only locally

### CODE REVIEW

File: frontend/src/app/modules/auth/login-page/login-page.component.ts [hunk 1]
```diff
@@ -88,7 +88,10 @@ export class LoginPageComponent implements OnInit {
     });
   }
 
-  public googleLogin = () => {
-    this.externalAuthService.signInWithGoogle();
+  public googleLogin = (event: FocusEvent) => {
+    event.preventDefault();
+    this.externalAuthService.signInWithGoogle().catch(() => {
+      this.snackbarService.openSnackBar('Error');
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/auth/login-page/login-page.component.ts]: Add more meaningful message please

File: frontend/src/app/modules/auth/register-form/register-form.component.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export class RegisterFormComponent implements OnInit {
           [
             Validators.required,
             Validators.pattern(
-              '^[а-яА-ЯёЁa-zA-Z\`\'][а-яА-ЯёЁa-zA-Z-\`\' ]+[а-яА-ЯёЁa-zA-Z\`\']?$'
+              "^[а-яА-ЯёЁa-zA-Z`'][а-яА-ЯёЁa-zA-Z-`' ]+[а-яА-ЯёЁa-zA-Z`']?$"
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/auth/register-form/register-form.component.ts]: Github actions

**Comment by aydevelop** [comment]
Fix

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Comment by aydevelop** [comment]
Please approval- merged conflicts

**Review by NickRost — DISMISSED** [review]

**Review by Jyuart — APPROVED** [review]

**Pull request merged**